templates/OperationResponseData.stencil

```diff
@@ -0,0 +1,7 @@
+guard let data = data else {
```

Comment:
This file should probably be named with `Snippet` at the end.

Reply:
Good point.